docs/docs/en/guide/task/sql.md

```diff
@@ -43,6 +43,12 @@ Log in to the bigdata cluster and use 'hive' command or 'beeline' or 'JDBC' and
 
 ![hive-sql](../../../../img/tasks/demo/hive-result.png)
 
+### Use Pre-SQL and Post-SQL
```

Comment:
It is seem this use case is not related to above, can we and the other section for above docs? I mean we should

```diff
+ ### Hive Table Create Example
+
- ### Create a Temporary Table in Hive and Write Data
+ #### Create a Temporary Table in Hive and Write Data
This example creates a temporary table `tmp_hello_world` in Hive and writes a row of data. Before creating a temporary table, we need to ensure that the table does not exist. So we use custom parameters to obtain the time of the day as the suffix of the table name every time we run, this task can run every different day. The format of the created table name is: `tmp_hello_world_{yyyyMMdd}`.
![hive-sql](../../../../img/tasks/demo/hive-sql.png)
- ### After Running the Task Successfully, Query the Results in Hive
+ #### After Running the Task Successfully, Query the Results in Hive
Log in to the bigdata cluster and use 'hive' command or 'beeline' or 'JDBC' and other methods to connect to the 'Apache Hive' for the query. The query SQL is `select * from tmp_hello_world_{yyyyMMdd}`, please replace `{yyyyMMdd}` with the date of the running day. The following shows the query screenshot:

 ![hive-sql](../../../../img/tasks/demo/hive-result.png)
```

Reply:
Thank you, I will modify it

Reply:
done.